tests/unit/test_coordinator_status.py

```diff
@@ -48,7 +48,7 @@ def __init__(self, framework: ops.Framework):
             nginx_config=NginxConfig("localhost", {}, {}),
             workers_config=lambda _: "worker config",
             resources_requests=lambda _: {"cpu": "50m", "memory": "100Mi"},
-            container_name="charm",
+            container_name="workload",
```

Comment:
@adhityaravi I think this should fix the failed unit tests that were having trouble with kubnernetes resource patching